refinery/ui/source/js/commons/data-sets/search-api.spec.js

```diff
@@ -68,7 +68,7 @@ describe('DataSet.search-api: unit tests', function () {
       'hl.simple.post': '%3C%2Fem%3E',
       'hl.simple.pre': '%3Cem%3E',
       q: _query,
-      qf: 'title%5E0.5+accession+submitter+text',
+      qf: 'title%5E0.5+accession+submitter+text+description',
```

Comment:
@flekschas : I was finding that not all DataSet description information was included in this text field. 

For example: 
`With "EdgeNGramming"`:
<img width="489" alt="screen shot 2017-05-04 at 2 43 18 pm" src="https://cloud.githubusercontent.com/assets/5629547/25720351/9d6828c4-30da-11e7-8ed6-7454fa2e1707.png">

and Normally:
<img width="476" alt="screen shot 2017-05-04 at 2 38 26 pm" src="https://cloud.githubusercontent.com/assets/5629547/25720376/af988dfe-30da-11e7-9776-3b7c3a1f03e7.png">

This approach is more resource intensive, but it doesn't seem too bad at first glance.
Using `117` DataSets:

EdgeNGram:
```
{
  "solr_home":"/vagrant/refinery/solr/",
  "version":"5.3.1 1703449 - noble - 2015-09-17 01:48:15",
  "startTime":"2017-05-04T18:54:01.44Z",
  "uptime":"0 days, 0 hours, 0 minutes, 46 seconds",
  "memory":"76.3 MB (%15.6) of 490.7 MB"}
```
Normal:
```
{
  "solr_home":"/vagrant/refinery/solr/",
  "version":"5.3.1 1703449 - noble - 2015-09-17 01:48:15",
  "startTime":"2017-05-04T18:43:10.319Z",
  "uptime":"0 days, 0 hours, 6 minutes, 0 seconds",
  "memory":"74.5 MB (%15.2) of 490.7 MB"}
```

I'm probably not educated enough on this topic to see any repurcussions? 

@ngehlenborg: Would we be okay with taking this resource hit for better search results?